library/dd-sdk-android-session-replay-material/src/test/kotlin/com/datadog/android/sessionreplay/material/forge/WireframeForgeryFactory.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.forge
+
+import com.datadog.android.sessionreplay.model.MobileSegment
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.ForgeryFactory
+
+internal class WireframeForgeryFactory : ForgeryFactory<MobileSegment.Wireframe> {
+    override fun getForgery(forge: Forge): MobileSegment.Wireframe {
+        return when (forge.anInt(min = 0, max = 2)) {
```

Comment:
minor: can be simplified just to `forge.aBool`

Reply:
I foresee more types later so I would rather keep it this way.